src/ApiProblem.php

```diff
@@ -256,7 +266,7 @@ protected function getDetail()
      *
      * @return string
      */
-    protected function getStatus()
+    public function getStatus()
```

Comment:
Same comment here: access is available via direct property access (`$problem->status`). I'll revert on merge.